laravel/app/Http/Controllers/ScheduleDataController.php

```diff
@@ -469,306 +380,156 @@ public function updateMilestones($id){
             ->orderBy('last_name')
             ->get();
 
-        $datas = compact('data1', 'data2', 'data3');
+        $datas = $data;
 
         return view('schedules.resident.milestone', compact('id', 'milestones', 'datas', 'anesthesiologists'));
     }
 
-	public function notifyResidentOverwrittenPreferences($toName, $toEmail, $residentName, $date, $overwrittenChoices)
+    public function notifyResidentOverwrittenPreferences($toName, $toEmail, $residentName, $date, $overwrittenChoices)
     {
 
-		$choice = "";
-		if($overwrittenChoices[0] == 1){
-			$choice = "1";
-		}
+        $choice = "";
+        if ($overwrittenChoices[0] == 1) {
+            $choice = "1";
+        }
 
-		if($overwrittenChoices[1] == 2){
-			$choice = $choice." 2";
-		}
+        if ($overwrittenChoices[1] == 2) {
+            $choice = $choice . " 2";
+        }
 
-		if($overwrittenChoices[2] == 3){
-			$choice = $choice." 3";
-		}
+        if ($overwrittenChoices[2] == 3) {
+            $choice = $choice . " 3";
+        }
 
-		$subject = 'REMODEL: Resident Preference '.$choice.' Overwritten for '.$date;
-		$body = "Resident $residentName has overwritten OR preferences  ".$choice." for ".$date.". New preferences are now viewable on REMODEL website.";
-		$heading = "Resident $residentName has overwritten OR preference ".$choice;
-        $data = array('name'=>$toName, 'heading'=>$heading, 'body'=>$body);
+        $subject = 'REMODEL: Resident Preference ' . $choice . ' Overwritten for ' . $date;
+        $body = "Resident $residentName has overwritten OR preferences  " . $choice . " for " . $date . ". New preferences are now viewable on REMODEL website.";
+        $heading = "Resident $residentName has overwritten OR preference " . $choice;
+        $data = array('name' => $toName, 'heading' => $heading, 'body' => $body);
 
-        Mail::send('emails.mail', $data, function($message) use ($toName, $toEmail, $subject) {
+        Mail::send('emails.mail', $data, function ($message) use ($toName, $toEmail, $subject) {
             $message->to($toEmail, $toName)->subject($subject);
             $message->from('OhioStateAnesthesiology@gmail.com');
         });
-		return true;
+        return true;
```

Comment:
It seems like this method can only return `true`. Unless I'm missing something, why bother returning anything?